test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/KieServerS2ISmokeIntegrationTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2017 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.cloud.integrationtests.smoke;
+
+import java.util.List;
+import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.Before;
+import org.junit.Test;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
+import org.kie.cloud.api.scenario.EmptyScenario;
+import org.kie.cloud.common.provider.KieServerClientProvider;
+import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
+import org.kie.server.api.model.KieContainerResource;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.api.model.definition.ProcessDefinition;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.QueryServicesClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KieServerS2ISmokeIntegrationTest extends AbstractCloudIntegrationTest<EmptyScenario> {
+
+    protected KieServicesClient kieServicesClient;
+    protected QueryServicesClient queryServicesClient;
+
+    private static final Logger logger = LoggerFactory.getLogger(KieServerS2ISmokeIntegrationTest.class);
+
+    private static final String KIE_CONTAINER_DEPLOYMENT = CONTAINER_ID + "=" + PROJECT_GROUP_ID + ":" + DEFINITION_PROJECT_NAME + ":" + DEFINITION_PROJECT_VERSION;
+    private static final String SOURCES_GIT_REPO_URL = "https://github.com/kiegroup/kie-cloud-tests.git";
```

Comment:
Could you please use OpenShift GIT provider for these sources? That way we can achieve better separation.